packages/webcomponents/src/api/Identity.ts

```diff
@@ -64,6 +68,23 @@ export class Owner implements Identity {
     this.title = owner.title;
     this.updated_at = owner.updated_at;
   }
+
+  // Getter method to return the payload in the desired shape
+  public get payload() {
```

Comment:
One small thing - we need to add `identification_number` to the payload getter for both Representative and Owner.  This is the property that posts the SSN for the data. 👍🏻